src/azure-cli/azure/cli/command_modules/databoxedge/tests/hybrid_2019_03_01/example_steps.py

```diff
@@ -0,0 +1,284 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=unused-argument
+
+from .. import try_manual
+
+
+# EXAMPLE: /Devices/put/DataBoxEdgeDevicePut
+@try_manual
+def step_device_create(test, rg, checks=None):
+    if checks is None:
+        checks = []
+    test.cmd('az databoxedge device create '
+             '--location "eastus" '
+             '--sku name="Edge" tier="Standard" '
+             '--name "{myDevice}" '
+             '--resource-group "{rg}"',
+             checks=checks)
+
+
+# EXAMPLE: /Devices/get/DataBoxEdgeDeviceGetByName
+@try_manual
+def step_device_show(test, rg, checks=None):
+    if checks is None:
+        checks = []
+    test.cmd('az databoxedge device show '
+             '--name "{myDevice}" '
+             '--resource-group "{rg}"',
+             checks=checks)
+
+
+# EXAMPLE: /Devices/get/DataBoxEdgeDeviceGetByResourceGroup
+@try_manual
+def step_device_list(test, rg, checks=None):
+    if checks is None:
+        checks = []
+    test.cmd('az databoxedge device list '
+             '--resource-group "{rg}"',
+             checks=checks)
+
+
+# EXAMPLE: /Devices/get/DataBoxEdgeDeviceGetBySubscription
+@try_manual
+def step_device_list2(test, rg, checks=None):
+    if checks is None:
+        checks = []
+    test.cmd('az databoxedge device list '
```

Comment:
the command with null value for `-g`?

Reply:
I checked with @changlong-liu before. He said it's by design. This case will be recognized as list by subscription.

Reply:
The intension is using -g "" to tell CLI runtime that this is a querying by subscription instead of "default resourcegroup"